src/object.d

```diff
@@ -529,6 +529,68 @@ bool __equals(T1, T2)(T1[] lhs, T2[] rhs)
     assert(a1 != a2);
 }
 
+void __move_post_blt(S)(ref S newLocation, ref S oldLocation) nothrow
+    if (is(S == struct))
+{
+    static foreach (memberName; __traits(allMembers, S))
+    {
+        static if (is(typeof(__traits(getMember, S, memberName)) == struct))
+        {
+            mixin("__move_post_blt(newLocation." ~ memberName ~ ", oldLocation." ~ memberName ~ ");");
```

Comment:
Can `__traits(getMember)` be used instead of the string mixin?

Reply:
Good call, fixed.